src/card.css

```
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: baseline;
+  padding: 32px 16px 16px;
+  height: 408px;
+  max-width: 200px;
```

```suggestion
  width: 200px;
```
Width should be fixed